src/app/app.component.ts

```diff
@@ -69,6 +74,15 @@ export class AppComponent implements OnInit {
       correlationId: CryptoUtils.createNewGuid(),
       piiLoggingEnabled: false
     }));
+
+    this.subscriptions.push(this.subscription1);
+    this.subscriptions.push(this.subscription2);
+  }
+
+  ngOnDestroy(): void {
+    if (this.subscriptions.length > 0) {
```

Comment:
Don't need to check length when using `forEach`, sice it is initialized to an empty array.

Reply:
aah, of course.

Reply:
done!